libs/isograph-react/src/core/IsographEnvironment.ts

```diff
@@ -125,14 +126,17 @@ export type DataId = string;
 
 export const ROOT_ID: DataId & '__ROOT' = '__ROOT';
 
-export type IsographStore = {
+export type DataLayer = {
```

Comment:
Let's clean up these names. I don't think we should use `Layer` for both `DataLayer` and `OptimisticLayer`. `OptimisticLayer` is a wrapper around `DataLayer`.